addons/calendar/models/calendar_event.py

```diff
@@ -95,6 +95,9 @@ def _default_partners(self):
                 partners |= self.env['res.partner'].browse(active_id)
         return partners
 
+    def _default_access_token(self):
+        return uuid.uuid4().hex
+
```

Comment:
Since you're not using any record data here, you can use the api.model annotation :)
```suggestion
    @api.model
    def _default_access_token(self):
        return uuid.uuid4().hex

```